docs/en/stack/ml-biases.asciidoc

```diff
@@ -0,0 +1,65 @@
+[[ml-biases]]
+=== Eliminating biases in {ml} processes
+
+{ml-cap} is meant to improve our everyday life in a wide palette of activities 
+across the globe. We use {ml} in science, healthcare, business, entertainment, 
+and many other fields. We make decisions based on {ml} results. It is therefore 
+crucial to keep in mind the possible biases that can corrupt results, endanger 
+fairness, and jeopardize the success of even the best-intended initiative. Some 
+of these social biases can make their way into the {ml} results more easily than 
+we might think.
+
+There are various possible biases that can negatively influence the result of a 
+{ml} analysis. *These biases typically are not present in the {ml} algorithms 
+that are used*, but rather exist in the datasets themselves or in the way the 
+results are interpreted. These issues are part of a greater question about the 
+ethics behind technology and there's no one solution for everyone. However, 
+there are some basic practices you can use to reduce these biases.
+
+
+[float]
+==== Biases in datasets
+
+Built-in biases exist in the data that we use to train {ml} models. For example,
+our choice of which data to collect or analyze unwittingly reflect our ideas 
```

Comment:
Not only what data is chosen to be collected, but what data is even available. This is a subtle difference, as a lot of training is done on available open datasets - namely people often rely on others to do the collection. I came across an interesting example recently - models for "repeat offenders" in the criminal justice system only train on - you guessed it - repeat offenders. Data is not collected for those who leave prison and act in accordance with the law. The problem here is both in the data collection (i.e. this data should be collected) and the data availability (in the absence of this data being collected, how can we address this).